### PR TITLE
noresults: allow messaging to be configured

### PR DESCRIPTION
**Pull request by benmcginnis**

### CODE REVIEW

File: src/ui/templates/results/noresults.hbs [hunk 1]
```diff
@@ -1,20 +1,32 @@
...
+        {{_config.noResultsMessage}}
+      {{else}}
+        Your search - <span class="yxt-NoResults-query">{{query}}</span> - did not match any answers we have.
+      {{/if}}
     </p>
     {{#if universalUrl}}
       <p class="yxt-NoResults-returnLinkWrapper">
-        Try refining your query or <a href="{{universalUrl}}" class="yxt-NoResults-returnLink">search all content.</a>
+        {{#if _config.noResultsTryUniversalMessage}}
+          {{{_config.noResultsTryUniversalMessage}}}
```
Comment (creotutar) [inline src/ui/templates/results/noresults.hbs:13]: why triple?

Comment (creotutar) [reply src/ui/templates/results/noresults.hbs:13]: i guess if someone adds html to this config option, they're not trying to sabotage the site

File: src/ui/templates/results/noresults.hbs [hunk 2]
```diff
@@ -1,20 +1,32 @@
 <div class="yxt-NoResults">
   <div class="yxt-NoResults-wrapper">
     <p class="yxt-NoResults-info">
-      Your search - <span class="yxt-NoResults-query">{{query}}</span> - did not match any answers we have.
+      {{#if _config.noResultsMessage}}
+        {{_config.noResultsMessage}}
```
Comment (creotutar) [inline src/ui/templates/results/noresults.hbs:5]: will a span work here?

**Review by creotutar — APPROVED** [review]
🚀 🚀 🚀 LGTM

File: src/ui/templates/results/noresults.hbs (hunk 1 above)
Comment (benmcginnis) [reply src/ui/templates/results/noresults.hbs:13]: yeah the purpose of this message is to prompt a user to visit the main universal results page, so it has to accept html

File: src/ui/templates/results/noresults.hbs (hunk 2 above)
Comment (benmcginnis) [reply src/ui/templates/results/noresults.hbs:5]: No, the query can't be passed in. Jon Hyler would execute me even if we're going to be responsible and pre-escape things, someone else might not.

**Review by tmeyer2115 — APPROVED** [review]

**Pull request merged**